plugins/scenesPageRememberStates/scenePageRememberStates.yml

```diff
@@ -0,0 +1,9 @@
+name: Scene Page Remember States
+description: Uses local storage to remember the state of the scene page detail panel nav bar and activate it on page load. Remembers collapse state of the divider.
+url:
+version: 0.1
+ui:
+  requires:
+    - stashUserscriptLibrary
```

Comment:
`stashUserscriptLibrary` is no longer part of the CommunityScripts repository. It was replaced with https://github.com/stashapp/CommunityScripts/tree/main/plugins/CommunityScriptsUILibrary, but it might be lacking some functionality. 

Needs a review.